tests/functional/TestSmokeTest/README.md

```diff
@@ -7,7 +7,44 @@ This functional test can be used on a system with at least 8GB of RAM.
 * Ansible
 * Hypervisor: VirtualBox or Libvirt/KVM
 
-## Setup 
+## Create VMs and run all the Tests
+
+ Just execute `run.sh` in this directory `tests/functional/TestSmokeTest`. It does the following:
+* creates 4 VMs using vagrant
+* builds heketi
+* starts heketi
+* executes every test in `./tests/heketi_test.go`
+* stops heketi
+* deletes heketi.db
+* destroys all 4 VMs
+
+```bash
+$ ./run.sh
+```
+
+Output will be shown by the logs on the heketi server.
+
+### To retain VMs after test
+If you wish to retain the setup after the tests, you can use the following command instead:
+
+```bash
+$ HEKETI_TEST_NO_CLEANUP="yes" ./run.sh
```

Comment:
Generally I think it is more preferable to have variables that are true/false on their face and don't include an implicit negation. To be more specific HEKETI_TEST_CLEANUP=(yes/no)  instead of HEKETI_TEST_NO_CLEANUP=(no/yes). Understanding that in the HEKETI_TEST_CLEANUP case Yes would be the default. This goes for the other variables that also have a _NO_ in the name.

This might make the code a tad more different than before but I think it is much easier to reason about and remember.

Reply:
Makes sense. Done.